src/kernel/panic.zig

```diff
@@ -90,6 +90,57 @@ const SymbolMap = struct {
     }
 };
 
+/// Copied from std.debug
+const StackIterator = struct {
+    first_address: ?usize,
+    fp: usize,
+
+    pub fn init(first_address: ?usize, fp: ?usize) StackIterator {
+        return StackIterator{
+            .first_address = first_address,
+            .fp = fp orelse @frameAddress(),
+        };
+    }
+
+    const fp_offset = if (builtin.arch.isRISCV())
+        2 * @sizeOf(usize)
+    else
+        0;
+
+    fn next(self: *StackIterator) ?usize {
+        var address = self.next_internal() orelse return null;
+
+        if (self.first_address) |first_address| {
+            while (address != first_address) {
+                address = self.next_internal() orelse return null;
+            }
+            self.first_address = null;
+        }
+
+        return address;
+    }
+
+    fn next_internal(self: *StackIterator) ?usize {
```

Comment:
The fact that there's a `next_internal` functions must mean that `next` is supposed to be public, but we can use this until the std lib is fixed (if it is going to be) 👍

Reply:
Sure, also still working on the python stuff :)